### DISCUSSION TITLE
Love this, good work!

### DISCUSSION THREAD

**Discussion by CriticalPoint** [NiyaShy/XB1ControllerBatteryIndicator#85] [General]
A couple notes...

I was reading about the update XML and I noticed that the XML states a version lower than the current release:
```
<?xml version="1.0" encoding="utf-8"?>
<XB1ControllerBatteryIndicator>
---->     <version>1.3.1.0</version>     <----
  <url>https://github.com/NiyaShy/XB1ControllerBatteryIndicator</url>
</XB1ControllerBatteryIndicator>
```
Given that the current release is v1.3.1.2, the updater wouldn't function - possibly just the xml needs an update 🙂

Next, I'd like to link to this from my repo [Automatically-Disable-Steam-Big-Picture-Mode-Guide-Button](https://github.com/CriticalPoint/Automatically-Disable-Steam-Big-Picture-Mode-Guide-Button) because it's a super cool tool and I use it daily, any issues with me doing that?

**Comment by NiyaShy** [comment]
Hey, thanks for the praise 😄

Concerning the XML file, it's kind of intentional that it still contains an older version number. My plan was that the current update checker only reacts to the first 3 numbers and ignores the 4th, thus only notifying about bigger code changes/new features. The 4th number level is intended for minor changes like new/updated languages, and v1.4 will get a new option so you can toggle that additional check.
But since the work on 1.4 has taken way longer than I originally thought, I've already started with using the 4th number for minor updates just so the releases can be better discerned.

And about your PS script, looks interesting, though I so far haven't had any issues with Steam reacting to the guide button.
